DataScience/IntroToPython/IntroToPythonPandas/blog.md

```diff
@@ -0,0 +1,215 @@
+
+Today, we will walk through some of the fundamentals of Pandas and use a variety of commands. This walktrough will use the Spyder IDE from [Anaconda](https://www.anaconda.com/products/individual
+). Once installed, go to your application dashboard, then open the new Anaconda3 folder and click on Spyder. Before we can do any work, we have to install pandas through our console. Some editors have basic packages pre-installed, but always check beforehand.
+
+A common way to install packages is:
+```
+pip install <package_name>
+```
+In Spyder: click on the Console in the bottom right corner and enter the following command below:
+```
+pip install pandas
+```
+
+Finally, we will need [data to work with](https://www.kaggle.com/benroshan/factors-affecting-campus-placement/data). Once downloaded, click show in folder so that you know its directory. If in a .zip, go into the .zip and copy the csv, go back to the original directory and paste.
+
+### To Do List
+
+#### 1. Install and Create Datasets (Done!)
+
+#### 2. Using Commands to Extract Data
+
+#### 3. Alter Our Datasets
+
+#### 4. Merge and Present Information
+
+## Extracting Our Data
+
+Congrats! We have our library installed and our data set to work with. Now, go into Spyder and in the left terminal hit enter, then type the following to have access to  pandas library:
+```
+import pandas as pd
+```
+The “import pandas” portion now includes the entire library for us to use. The “as pd” portion is to use pd as a shortcut when calling the library functions. So now we don’t type out pandas.(function_name) everytime, but instead pd.(function_name). Yay Efficiency! Our first line of code will be to get our data set into a DataFrame object (so find your data!) Enter the file location of your .csv file into the pd.read_csv(“file”) function like so:
+```py
+data = pd.read_csv("C://Users//<user_name>//Downloads//factors-affecting-campus-placement//Placement_Data_Full_Class.csv")
+```
+
+On Spyder, there’s a tab called *Variable Explorer* in the middle right. Click on it, and you should see a new DataFrame object of size (215,15).
+
+|Name|Type|Size|Value
+|:-----:|------:|:-----:|------:|
+|data|DataFrame|(215,15)|Column names: sl_no, gender, ssc_p, ssc_b, hsc_p, hsc_b, hsc_s, degree...|
+
+A DataFrame is just like an Excel table that we can directly edit using our code.
+When creating a DataFrame that is not necessarily imported with read_csv, you can directly build a Dataframe in the following format:
+```py
+alt = pd.DataFrame({'col A':[1,2], 'col B':[3,4]})
+```
+Result: 
+
+|Name|Type|Size|Value
+|:-----:|------:|:-----:|------:|
+|alt|DataFrame|(2,2)|Column names: A,B |
+
+We have our data with us, but it’s very large, so we might be confused as to what to do with it right now... Here are some examples of the commands to use:
+- **data.head():** returns top 5 rows
```

Comment:
Instead of having bullet points of commands, you should demonstrate each one individually, documenting the inputs/outputs to make sure the beginners understand what each one does.